composer.json

```diff
@@ -13,7 +13,14 @@
     "require": {
         "php": "^5.5 || ^7.0"
     },
+    "repositories": {
+        "client-common": {
+            "type": "vcs",
+            "url": "git@github.com:fbourigault/client-common"
+        }
+    },
     "require-dev": {
+        "php-http/client-common": "dev-plugin-client-factory",
```

Comment:
just so we don't forget: we have to remove this before merging, when client-common is merged and released